src/lib/stitching/causality/__tests__/stitching.test.ts

```diff
@@ -157,4 +155,136 @@ describe("causality/stitching", () => {
       },
     })
   })
+
+  describe("auctionsLotStandingConnection", () => {
+    it("stitches auctionsLotStandingConnection under Me", async () => {
+      const allMergedSchemas = await incrementalMergeSchemas(schema, 1)
+
+      const query = gql`
+        {
+          me {
+            auctionsLotStandingConnection {
+              edges {
+                node {
+                  rawId
+                }
+              }
+            }
+          }
+        }
+      `
+
+      // Mock the resolvers for just a user's lot standings so we can see the
+      // stitched SaleArtwork data inside.
+      // this is needed without it we get [GraphQLError: Variable "$_v0_userId" got invalid value undefined; Expected non-nullable type ID! not to be null.]
+      addMockFunctionsToSchema({
+        schema: allMergedSchemas,
+        mocks: {
+          Query: () => ({
+            me: () => {
+              return {
+                internalID: "foo",
+              }
+            },
+          }),
+        },
+      })
+
+      const result = await graphql(allMergedSchemas, query, {}, {})
+
+      expect(result).toEqual({
+        data: {
+          me: {
+            auctionsLotStandingConnection: {
+              edges: [
+                { node: { rawId: "Hello World" } },
+                { node: { rawId: "Hello World" } },
+              ],
+            },
+          },
+        },
+      })
+    })
+
+    describe("when sale artworks are missing", () => {
```

Comment:
These two just test the resolver to check if it behaves correctly when `saleArtworkRootLoader` throws error(s). I first tried to combine these but realized separating stitching and resolvers tests make things much simpler. Is there any other edge case that needs to be covered?